docs/source/faq.rst

```diff
@@ -190,6 +190,20 @@ You can also find the failed trials by checking the trial states as follows:
     1,TrialState.COMPLETE,1269,...,1,
 
 
+What happens when I dynamically alter a search space?
+-----------------------------------------------------
+
+Since parameters search spaces are specified in each call to the suggestion API, e.g.
+:func:`~optuna.trial.Trial.suggest_uniform` and :func:`~optuna.trial.Trial.suggest_categorical`,
```

Comment:
```suggestion
:func:`~optuna.trial.Trial.suggest_uniform` and :func:`~optuna.trial.Trial.suggest_int`,
```

I suggested the above change because the current implementation actually doesn't allow to alter the search space of categorical parameters dynamically (cf. https://github.com/optuna/optuna/issues/372#issuecomment-479988730 ).

Reply:
Thanks for the correction. Let me apply the suggestion.